### PR TITLE
GHC-90: Improve Second Vitals Form

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/bahmniforms/second_vital.json [hunk 1]
```diff
@@ -1,89 +1,83 @@
...
+    {
+      "locale": "es",
+      "labels": {},
+      "concepts": {
+        "DIASTOLIC_4": "Diastólica",
+        "SYSTOLIC_3": "Sistólica",
+        "BLOOD_PRESSURE_2": "Presión sanguínea",
+        "TEMPERATURE_5": "La temperatura",
+        "OXYGEN_SATURATION_7": "Saturación de oxígeno",
+        "RESPIRATORY_RATE_6": "La frecuencia respiratoria",
```
Comment (rbuisson) [inline configuration/bahmniforms/second_vital.json]: Should it rather be "Frequencia respiratoria" without "La"?

File: configuration/bahmniforms/second_vital.json [hunk 2]
```diff
@@ -1,89 +1,83 @@
...
+        "PULSE_1": "Legumbres"
```
Comment (rbuisson) [inline configuration/bahmniforms/second_vital.json]: Is this the right translation?
Ot should be more "Pulso" I believe.

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/bahmniforms/second_vital.json (hunk 1 above)
Comment (reagan-meant) [reply configuration/bahmniforms/second_vital.json]: I followed this [here](https://docs.google.com/spreadsheets/d/1kcStR8bC2W8eVJX2jhVdQBm5GSGb3bt59DP5f8jMyuY/edit#gid=0)

Comment (rbuisson) [reply configuration/bahmniforms/second_vital.json]: Changed in the file.

File: configuration/bahmniforms/second_vital.json (hunk 2 above)
Comment (reagan-meant) [reply configuration/bahmniforms/second_vital.json]: Done

**Pull request merged**